content/applications/finance/expenses.rst

```diff
@@ -79,88 +93,112 @@ Manually create a new expense
+     update in real time as the :guilabel:`Total` or :guilabel:`Quantity` is updated.
 
+- :guilabel:`Employee`: Using the drop-down menu, select the employee this expense is for.
 - :guilabel:`Paid By`: Click the radio button to indicate who paid for the expense and should be
   reimbursed. If the employee paid for the expense (and should be reimbursed) select
   :guilabel:`Employee (to reimburse)`. If the company paid directly instead (e.g. if the company
-  credit card was used to pay for the expense) select :guilabel:`Company`.
+  credit card was used to pay for the expense) select :guilabel:`Company`. Depending on the expense
+  category selected, this field may not appear.
+- :guilabel:`Bill Reference`: If there is any reference text that should be included for the
+  expense, enter it in this field.
 - :guilabel:`Expense Date`: Using the calendar module, enter the date the expense was incurred. Use
   the :guilabel:`< (left)` and :guilabel:`> (right)` arrows to navigate to the correct month, then
   click on the specific day to enter the selection.
-- :guilabel:`Bill Reference`: If there is any reference text that should be included for the
-  expense, enter it in this field.
 - :guilabel:`Account`: Select the expense account that this expense should be logged on from the
   drop-down menu.
-- :guilabel:`Employee`: Using the drop-down menu, select the employee this expense is for.
 - :guilabel:`Customer to Reinvoice`: If the expense is something that should be paid for by a
-  customer, select the customer that will be invoiced for this expense from the drop-down menu. For
-  example, if a customer wishes to have an on-site meeting, and agrees to pay for the expenses
-  associated with it (such as travel, hotel, meals, etc.), then all expenses tied to that meeting
-  would indicate that customer as the :guilabel:`Customer to Reinvoice`.
-- :guilabel:`Analytic Account`: Select the account the expense should be written against from the
-  drop-down menu.
-- :guilabel:`Company`: If multiple companies are set-up, select the company this expense should be
-  filed for from the drop-down menu. If there is only one company, this field will be automatically
-  populated.
+  customer, select the :abbr:`SO (Sales Order)` and customer that will be invoiced for this expense
+  from the drop-down menu. All sales orders in the drop-down menu list both the :abbr:`SO (Sales
+  Order)` as well as the company the sales order is written for, but after the expense is saved, the
+  customer name disappears and only the :abbr:`SO (Sales Order)` is visible on the expense.
+
+  .. example::
+     A customer wishes to have an on-site meeting for a custom garden (design and installation) and
+     agrees to pay for the expenses associated with it (such as travel, hotel, meals, etc.). All
+     expenses tied to that meeting would indicate the *sales order* for the custom garden (which
+     also references the customer) as the :guilabel:`Customer to Reinvoice`.
```

Comment:
ok, I initially did because that's a weird thing to reference - you would expect the customer name to be the reference, not a sales order, but I understand- and agree, and changed it =)